docs/advanced_topics/images/custom_image_model.rst

```diff
@@ -7,7 +7,10 @@ Custom image models
 The ``Image`` model can be customised, allowing additional fields to be added
 to images.
 
-To do this, you need to add two models to your project:
+For projects started from Wagtail 2.9, customisable ``Image`` and ``Rendition`` models are located in the ``media`` app
```

Comment:
Would be good to add a similar note for [custom document models](https://docs.wagtail.io/en/stable/advanced_topics/documents/custom_document_model.html)